pkg/featuregate/feature_gate.go

```diff
@@ -98,7 +99,7 @@ type MutableFeatureGate interface {
 	FeatureGate
 
 	// AddFlag adds a flag for setting global feature gates to the specified FlagSet.
-	AddFlag(fs *pflag.FlagSet)
+	AddFlag(fs *flag.FlagSet, flagName string)
```

Comment:
Is there a particular reason why you replaced `pflag` with `flag`?

Reply:
all the other flags in `server/embed/config.go` uses `flag` instead of `pflag`

Reply:
That is good to know. Thanks for your answer. I would think that using `pflag` is better as it is POSIX-compliant, but I do see that across the project, we're using `flag` for flagsets. We still parse flags using Cobra, which makes them POSIX-compliant.